src/containers/Tenant/utils/schema.ts

```diff
@@ -272,6 +281,7 @@ const pathTypeToChildless: Record<EPathType, boolean> = {
     [EPathType.EPathTypeSubDomain]: false,
     [EPathType.EPathTypeTableIndex]: false,
     [EPathType.EPathTypeExtSubDomain]: false,
+    [EPathType.EPathTypeStreamingQuery]: false,
```

Comment:
Can streaming query have children? It seems, they don't. Please, ensure it with streaming query developer